install_config/install/prerequisites.adoc

```diff
@@ -461,16 +461,17 @@ properly when the network interface for {product-title} has `NM_CONTROLLED=no`.
 ==== Required Ports
 
 The {product-title} installation automatically creates a set of internal
-firewall rules on each host using `iptables`. However, if your network
-configuration uses an external firewall, such as a hardware-based firewall, you
-must ensure infrastructure components can communicate with each other through
-specific ports that act as communication endpoints for certain processes or
-services.
+firewall rules on each host using
+xref:../../admin_guide/iptables.adoc#overview[iptables]. However, if your
+network configuration uses an external firewall, such as a hardware-based
+firewall, you must ensure infrastructure components can communicate with each
+other through specific ports that act as communication endpoints for certain
+processes or services.
 
 [NOTE]
 ====
 While iptables is the default firewall, firewalld is recommended for new
-installations. See the xref:../../admin_guide/iptables.adoc#overview[iptables section of the Cluster administration guide] for more information.
+installations. You can enable firewalld by setting `os_firewall_use_firewalld=true` in xref:../../install_config/install/advanced_install.adoc#configuring-ansible[the Ansible inventory file].
```

Comment:
Can I suggest linking closer to the firewall section instead of the overall configuration section?

https://docs.openshift.com/container-platform/3.7/install_config/install/advanced_install.html#advanced-install-configuring-firewalls

Reply:
@bfallonf, I always want to hear suggestions, and thank you for pointing out a better link! I'll update the topic.